engine/src/main/java/com/redhat/cloud/notifications/processors/slack/SlackRouteBuilder.java

```diff
@@ -1,90 +0,0 @@
-package com.redhat.cloud.notifications.processors.slack;
-
-import org.apache.camel.CamelExchangeException;
-import org.apache.camel.builder.RouteBuilder;
-import org.eclipse.microprofile.config.inject.ConfigProperty;
-
-import javax.enterprise.context.ApplicationScoped;
-import javax.inject.Inject;
-
-import java.io.IOException;
-
-import static com.redhat.cloud.notifications.Constants.API_INTERNAL;
-import static org.apache.camel.LoggingLevel.ERROR;
-import static org.apache.camel.LoggingLevel.INFO;
-
-@ApplicationScoped
-public class SlackRouteBuilder extends RouteBuilder {
-
-    public static final String REST_PATH = API_INTERNAL + "/slack";
-    public static final String SLACK_OUTGOING_ROUTE = "slack-outgoing";
-    public static final String SLACK_INCOMING_ROUTE = "slack-incoming";
-
-    private static final String SLACK_DIRECT_ENDPOINT = "direct:slack";
-    private static final String EXCEPTION_DIRECT_ENDPOINT = "direct:exception";
-    private static final String ERROR_MSG = "Slack message sending failed [orgId=${exchangeProperty.orgId}, " +
-            "historyId=${exchangeProperty.historyId}, webhookUrl=${exchangeProperty.webhookUrl}, " +
-            "channel=${exchangeProperty.channel}]\n${exception.stacktrace}";
```

Comment:
Does the new code include the `channel` in the log entry? I can't find it.

Reply:
No you are right, I refactored it a little bit to add it on last commit